packages/website/src/components/RecruitmentModal/RecruitmentModal.tsx

```diff
@@ -36,8 +40,16 @@ export const RecruitmentModal = ({ isOpen, config, onClose, modalType }: Recruit
         boxShadow="large"
         textAlign="center"
       >
+        <IconButton
+          icon={<SolidCloseIcon />}
+          aria-label="Close modal button"
+          variant="ghost"
+          style={{ position: 'absolute', top: 12, right: 12 }}
```

Comment:
❓ Do you need the `style` attribute here? Cannot it be achieved by passing those style properties as props?